udata/tests/api/test_organizations_api.py

```diff
@@ -637,6 +637,35 @@ def test_suggest_organizations_homonyms(self, api):
         for suggestion in response.json:
             assert suggestion['name'] == 'homonym'
 
+    def test_suggest_organizations_acronym(self, api):
+        '''Should suggest organizations based on acronym'''
+
+        for i in range(3):
+            OrganizationFactory(
+                name=faker.word(),
+                acronym=f'TEST{i}' if i % 2 else faker.word(),
+                metrics={"followers": i})
+        max_follower_organization = OrganizationFactory(
+            name=faker.word(),
+            acronym='TEST4',
+            metrics={"followers": 10}
+        )
+        response = api.get(url_for('api.suggest_organizations'),
+                           qs={'q': 'tEsT', 'size': '5'})
+        assert200(response)
+
+        assert len(response.json) <= 5
```

Comment:
Shouldn't be `len(response.json)` precisely defined?

Reply:
I had the same concern when I was first looking at the existing tests for inspiration. As far as I understand the idea, we assume that the faker may return something that will match, that's why this `assert` is not strict. In case we use 'UDATA' as acronym, we can probably make it strict here. Should I make this change?

Reply:
Mmm indeed, we should probably clean other tests to make the assert strict. And yes please you can change for 'UDATA' here, to prevent adding more precarious tests! 🙏